beta/examples/tensorflow/classification/main.py

```diff
@@ -115,12 +115,26 @@ def load_checkpoint(model, ckpt_path):
     return None
 
 
-def resume_from_checkpoint(model, compression_ctrl, ckpt_path, steps_per_epoch):
+def get_scheduler_state(num_steps, steps_per_epoch, config):
```

Comment:
Shouldn't it be part of the scheduler API? As for me it looks complicated for the user to implement this. Moreover, this code looks identical for all samples.

IMO, it's a scheduler responsibility to create a scheduler state from some user parameters (e.g. num_steps, steps_per_epoch)

Reply:
It is a temporary solution. We are planning to save and load the scheduler's state to the checkpoint in the future. It will be implemented as separate PR.

Reply:
@andrey-churkin I support your plan, but anyway, could you create a temporal helper function that creates the scheduler state by input parameters.

Reply:
Done